server/zoom/agent/util/helpers.py

```diff
@@ -136,3 +139,31 @@ def cap_hostname(host):
     server = '.'.join([_split[0].upper()] + _split[1:])
     logging.debug('Converted: {0} to {1}'.format(host, server))
     return server
+
+def create_temporary_znode(zk_connection, zk_path, hostname):
+    """
+    Function to create a znode with a UUID to test sentinel -> zookeeper
+    connectivity. Record the hostname in the znode for debugging if one
+    gets left behind
+    :type zk_connection: KazooClient
+    :type zk_path: str
+    :return: bool
+    """
+    _log = logging.getLogger('util.create_temporary_znode')
+    _path = os.path.join(zk_path, str(uuid.uuid4()))
+    _log.debug('Using {0} as path'.format(_path))
+    _ret = False
+
+    try:
+        zk_connection.create(_path, hostname, makepath=True)
+        zk_connection.delete(_path)
+        _ret = True
+    except NoNodeError, ex:
+        _log.critical('Unable to create temporary zk node at {0}: {1}'.format(_path, ex))
```

Comment:
Well, kazoo will also throw a ```NoNodeError``` if it can't create due to part of the zpath not existing, though with makepath=True being passed that should not happen, so updating.